lib/entities/main_actions.dart

```diff
@@ -58,7 +58,7 @@ class MainActions {
           break;
         case WalletType.monero:
           if (viewModel.isEnabledBuyAction) {
-            if (DeviceInfo.instance.isMobile) {
+            if (ResponsiveLayoutUtil.instance.isMobile) {
```

Comment:
revert to Device Info, as the in-app webview package is not supported except by iOS and Android